volume/volume_windows.go

```diff
@@ -199,3 +205,25 @@ func validateStat(fi os.FileInfo) error {
 	}
 	return nil
 }
+
+func validateAbsolute(p string) error {
```

Comment:
It seems to me that this validation should be context sensitive. I.e. we should only perform Windows validation when handling mounts for Windows containers, and we should only perform Linux validation when handling mounts for Linux containers.

In the latter case, it would be nice to reuse the existing Linux validation code rather than rewrite it here, although of course that means the shared Linux code would need to use path instead of filepath.